test/integration/components/tooltip.js

```diff
@@ -61,7 +61,21 @@ context('Tooltip', function() {
         .get('@hook')
         .contains(model.id)
         .as('button')
-        .trigger('mouseover');
+        .trigger('pointerover');
+
+      cy
```

Comment:
Add additional test to make sure the tooltips open when the `pointerdown` event is fired on the element.